tests/kernel/run_ltp.pm

```diff
@@ -266,6 +266,7 @@ sub run {
         );
 
         script_run('ps axf');
+        script_run('netstat -ap');
```

Comment:
You're right (I've read that article as well), I'll correct that. But quite a lot of LTP networking scripts require commands from net-tools. 8 use netstat, and there is even one whole [netstat test](https://github.com/linux-test-project/ltp/blob/master/testcases/network/tcp_cmds/netstat/netstat01) :-(.